src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -36,6 +38,20 @@ export const addAggregatorAndDestinationToMemo = ({
     'expected finalAssetLimitWithManualSlippage to be a positive amount',
   )
 
+  const minimumPrecision = 5
+  const endingExponential = finalAssetPrecision - minimumPrecision
+
+  // The uniswap aggregator expects this amount to be an exponent, we need to add two numbers at the end which are used at exponents in the contract
+  // We trim 10 of precisions to make sure the THORChain parser can handle the amount without precisions and rounding issues
+  // If the finalAssetPrecision is under 5, the THORChain parser won't fail and we add one exponent at the end so the aggregator contract won't multiply the amount
+  const finalAssetLimitWithTwoLastNumbersAsExponent =
+    finalAssetPrecision <= 5
```

Comment:
As explained above, this will effectively never get hit unless we use 6 inclusive (also let's make sure to reuse the constant above here instead of magic numbers)

Reply:
Changed it to 6 inclusive